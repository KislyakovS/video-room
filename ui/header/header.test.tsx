import { render } from '@testing-library/react';

import Header from './header';

describe('<Header>', () => {
    it('snapshot', () => {
        const { container } = render(<Header
            leftSide='Left Side'
            center='Center'
            rightSide='Right Side'
        />);

        expect(container).toMatchSnapshot();
    });
});