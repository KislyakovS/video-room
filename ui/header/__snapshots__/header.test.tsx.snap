// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> snapshot 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="left"
    >
      Left Side
    </div>
    <div
      class="center"
    >
      Center
    </div>
    <div
      class="right"
    >
      Right Side
    </div>
  </header>
</div>
`;
